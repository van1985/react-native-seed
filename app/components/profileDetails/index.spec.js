import React from 'react';
import renderer from 'react-test-renderer';
import ProfileDetails from './index';

const user = {
  name: 'John',
  tech: 'UI Technology',
  email: 'john.smith@globant.com',
  city: 'Mar del Plata',
  country: 'Argentina',
};

test('renders correctly', () => {
  const tree = renderer.create(<ProfileDetails
    name={user.name}
    tech={user.tech}
    email={user.email}
    city={user.city}
    country={user.country}
  />).toJSON();
  expect(tree).toMatchSnapshot();
});
