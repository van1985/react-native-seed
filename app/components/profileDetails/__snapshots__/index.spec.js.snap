// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={undefined}
>
  <View
    style={
      Object {
        "backgroundColor": "#333333",
        "height": 200,
      }
    }
  >
    <View
      style={
        Object {
          "display": "flex",
          "flexDirection": "row",
          "paddingLeft": 30,
          "paddingTop": 40,
        }
      }
    >
      <View>
        <View
          accessibilityComponentType={undefined}
          accessibilityLabel={undefined}
          accessibilityTraits={undefined}
          accessible={true}
          collapsable={undefined}
          hitSlop={undefined}
          isTVSelectable={true}
          nativeID={undefined}
          onLayout={undefined}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#fff",
              "borderColor": "rgba(0,0,0,0.2)",
              "borderRadius": 50,
              "borderWidth": 1,
              "height": 100,
              "justifyContent": "center",
              "opacity": 1,
              "width": 100,
            }
          }
          testID={undefined}
          tvParallaxProperties={undefined}
        >
          <Image
            source={
              Object {
                "testUri": "../../../app/assets/silhouette200.png",
              }
            }
            style={
              Object {
                "borderRadius": 50,
                "height": 100,
                "width": 100,
              }
            }
          />
        </View>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "flex": 1,
            "justifyContent": "center",
            "paddingLeft": 10,
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#ffffff",
              },
              Object {
                "fontSize": 22,
              },
              Object {
                "paddingBottom": 5,
              },
            ]
          }
        >
          John
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#ffffff",
              },
              Object {
                "fontSize": 18,
              },
              Object {
                "paddingBottom": 5,
              },
            ]
          }
        >
          UI Technology
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#ffffff",
              },
              Object {
                "fontSize": 18,
              },
              Object {
                "paddingBottom": 5,
              },
            ]
          }
        >
          john.smith@globant.com
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#ffffff",
              },
              Object {
                "fontSize": 18,
              },
            ]
          }
        >
          Mar del Plata
          , 
          Argentina
        </Text>
      </View>
    </View>
  </View>
</View>
`;
